pkg/cmd/pr/view/view.go

```diff
@@ -54,6 +53,9 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
     	`),
 		Args: cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			// support `-R, --repo` override
```

Comment:
Is fixing this follow up work?

Reply:
No, this is not a TODO: it describes that the line below is in service of the `--repo` override. It already works!

Since `factory.BaseRepo` is being overridden in a Cobra "PreRun" hook, we need BaseRepo to be respected (and copied over to the Options object) in "RunE" since that runs after the hooks. This is not super evident so I've tried to explain via a comment, but I see now that it might have not been as clear